requirements.in

```diff
@@ -0,0 +1,11 @@
+black==21.7b0
```

Comment:
Going to hold off on that for now, but will explore this idea for the future.

Reply:
> TL;DR: I'm ok with leaving this as-is, but I want to make sure that I'm properly communicating why pip-compile isn't sufficient for the long-term.

Noted, and I am aware of the limitations of pip-compile that you mentioned (MozPhab has a similar but slightly more complex situation, and there I've solved that with simple docker containers to generate those files for each platform). I don't necessarily see this issue as needing another library and additional config files for a long term solution, but it's certainly something to consider.

Keep in mind that the requirements.txt file is mostly needed for CircleCI purposes, and also for local development. `mots` itself only has two requirements which are included in `setup.cfg` and does not require any lock files or anything like that. For CircleCI we always know ahead of time which platform we'll be running tests on.

Reply:
Another minor comment here is that the only requirement that is python-version-specific for packaging `mots`, is `importlib-resources`. It's definitely not an unimportant part of the code (since it will be used when exporting to rst) but I'd like to find a cleaner solution to fetch templates in the future, so this conditional dependency may be removed in the future.

Reply:
Filed a [bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1753756) for this. The requirements files will need to be fixed before merging/deploying this since tests will actually fail on Python 3.7 as things stand now.